recipes/slearn/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "slearn" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+  fn: slearn-0.2.4.tar.gz
+  url: https://files.pythonhosted.org/packages/f8/dd/3bd36708e78802bcab553e7b4bfc15a4a433e26fbe5f5911b1f2533fe8cd/slearn-0.2.4.tar.gz
+  sha256: e58e210667b3f96d6ec494778146708daaf3d4cdd318a320694c93db30b4743d
+
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
```

Comment:
Under `build` it looks like you only need `python >=3.6`, `pip` and `numpy`. The rest are runtime requirements not needed for the building of the package.

Reply:
Thanks! Modified.